binderhub/repoproviders.py

```diff
@@ -5,18 +5,21 @@
 control services and providers.
 
 """
-from datetime import datetime
+from datetime import timedelta
 import json
 import os
 import time
 
+from prometheus_client import Gauge
+
 from tornado import gen
 from tornado.httpclient import AsyncHTTPClient, HTTPError
 from tornado.httputil import url_concat
 
 from traitlets import Dict, Unicode, default
 from traitlets.config import LoggingConfigurable
 
+GITHUB_RATE_LIMIT = Gauge('binderhub_github_rate_limit', 'GitHub rate limit remaining')
```

Comment:
I think this should be named 'binderhub_github_rate_limit_remaining' so it's clear it's the remaining requests. Shouldn't rely solely on the descripton for that info.